MicrosoftTeams.schema.json

```diff
@@ -805,6 +805,24 @@
       },
       "description": "When a group install scope is selected, this will define the default capability when the user installs the app",
       "additionalProperties": false
+    },
+    "configurableProperties": {
+      "type": "array",
+      "description": "A list of tenant configurable properties for an app",
+      "maxItems": 9,
+      "items": {
+        "enum": [
+          "name",
+          "shortDescription",
```

Comment:
Same with required section below we are using the top level grouping... of "description"